website/docs/cookbooks/ide/intellij.md

```diff
@@ -3,7 +3,7 @@ title: IntelliJ IDEA setup
 sidebar_position: 11
 ---
 
-import {ChainedSnippets} from "../../src/components/MarkdownComponents.js";
+import {ChainedSnippets} from "../../../src/components/MarkdownComponents.js";
```

Comment:
Would it make sense to join the Intellij cookbooks into one?

Reply:
we could, but that would make it complex enough to make it a guide.